test/e2e/ps_test.go

```diff
@@ -425,6 +425,31 @@ var _ = Describe("Podman ps", func() {
 		Expect(session.OutputToString()).To(ContainSubstring("0.0.0.0:2000-2006"))
 	})
 
+	It("podman ps test with multiple port range", func() {
```

Comment:
(Not sure if one already exists, didn't look) Please consider adding a small/quick negative test to confirm podman throws an error if an overlapping range is given.  For example `-p 1000-2000:2000-3000` with `-p 1999-2999:3001-4001` and `-p 3000-4000:4001:5001`.  Reason being, it would be disastrous if something like this slipped through, creating a difficult to debug ambiguity for users.

Reply:
Sure, added a test like that. :+1: